libr/core/cmd.c

```diff
@@ -153,6 +153,22 @@ static const char *help_msg_star[] = {
 	NULL
 };
 
+static const char *table_help[] = {
+	"Usage:", ",[/] [file]", "# load table data",
+	",", "", "display table",
+	".", "$foo", "aflt > $foo (files starting with '$' are saved in memory)",
+	". ", " [table-query]", "filter and print table. See ,? for more details",
+	"..", " file.csv", "load the csv file into a table",
```

Comment:
I think this is not implemented.

Reply:
Correct. r_table_from_csv is not implemented and this command just tries to guess the format to load, which from my tests it seems to not work very reliabiliy. ill write tests for this and fix the behaviour